### PR TITLE
Request COE URL

### PR DESCRIPTION
**Pull request by jerekshoe**
## Description
If the users COE request status is either 'available' or 'eligible', they should have the ability to download their COE as a PDF. We need to make the request to get the URL for this PDF and set the href in the appropriate places to the returned URL.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#35595

## Definition of done
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

File: src/applications/lgy/coe/form/containers/introduction-content/COEIntroPageBox.jsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ const COEIntroPageBox = props => {
   if (props.coe.status) {
     switch (props.coe.status) {
       case COE_ELIGIBILITY_STATUS.available:
-        return <COEAvailable />;
+        return <COEAvailable downloadURL={props.downloadURL} />;
```
Comment (cohnjesse) [inline src/applications/lgy/coe/form/containers/introduction-content/COEIntroPageBox.jsx:14]: This is a really small thing, and not worth holding the PR up for, but we could destructure these props to make this a little cleaner, thoughts?

**Review by cohnjesse — APPROVED** [review]
Approved with one comment

File: src/applications/lgy/coe/form/containers/introduction-content/COEIntroPageBox.jsx (hunk 1 above)
Comment (jerekshoe) [reply src/applications/lgy/coe/form/containers/introduction-content/COEIntroPageBox.jsx:14]: Yeah I honestly thing that this is valid in a few other components as well. I'd like to refactor those bits when I come back to do error handling

Comment (cohnjesse) [reply src/applications/lgy/coe/form/containers/introduction-content/COEIntroPageBox.jsx:14]: @jerekshoe For sure, lets make a ticket for refactoring this and we can come back to it later

**Pull request merged**